2020-02-Spring/2_linked_lists/README.md

```diff
@@ -126,31 +147,117 @@ Go to [Top](#top)
 <!-- Don't remove -->
 <a name="s2"/>
 
-### 2. SOLUTION 2 TODO :bug:
-
-Source: TODO :bug:
+### 2. Remove Zero Sum Consecutive Nodes from Linked List
 
 #### Naive/Simple Solution
+The idea is to locate the sublist of nodes that sum to 0 and de-list them.<br>
+- Starting at the edging node, find lower-bound and upper-bound node of the sublist that sum to 0.
+- if found, de-list nodes from [LBound, UBound], including by re-link PreLBound->next = UBound->next;
+- If not found, move forward and check the next node.
+For every i-th node of N nodes, the algorithm iterates from node i-th to the last node for the total of N*(N-i) times.
```

Comment:
This bullet does not explain the relationship between inner loop complexity to search for a zero sum (time complexity `O(N - i)`) and the outer loop (`O(N)`). Recommend removing this bullet.
```suggestion
```